build.ps1

```diff
@@ -56,6 +56,11 @@ param (
 
 . "$PSScriptRoot\build\common.ps1"
 
+If (-Not $SkipDelaySigning)
```

Comment:
Should this be something that configure maybe does? 

But I get why it belongs here as this is the place where the decision to skip delay signing is.

Reply:
Do you mean in addition to or instead of build.ps1?

This option is a build-specific option not a repo-wide configuration option, so it makes sense to be in build.ps1.  Granted, this option may not get much usage (like configure.ps1), but that's OK.

Reply:
I meant instead. 

I just floating the idea and I think your reasoning makes sense.